pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2757,6 +2851,34 @@
       %-  =<  request-is-logged-in:authentication
           (per-server-event [eny *duct now rof] server-state.ax)
       %*(. *request:http header-list ['cookie' u.cookies]~)
+    ::
+        [%cache @ @ ~]
+      ?~  aeon=(slaw %ud i.t.tyl)        [~ ~]
+      ?~  url=(slaw %t i.t.t.tyl)        [~ ~]
+      ?~  entry=(~(get by cache) u.url)  [~ ~]
+      ?.  =(u.aeon aeon.u.entry)         [~ ~]
+      ?~  val.u.entry                    [~ ~]
+      =*  val  u.val.u.entry
+      ?:  ?=(%payload -.body.val)
+        ``noun+!>(`cache-entry`val)
+      ::  body is a scry path; scry it to get a payload
+      ::
+      =/  res=(unit mime)  (scry-to-mime rof [our %base da+now] path.body.val)
```

Comment:
This doesn't look right to me.  It hard-codes the `%base` desk as the location of the file to be scry'd out, but most use-cases for this will have files stored in other desks.

This suggests the representation of the scry path passed into Eyre should contain a desk and case (it should be possible to tell Eyre to serve an old version of a file too -- no reason not to afaik).  Now we're almost at a `$beam`, just missing `ship`.

So I think we should change the interface here to accept a fully qualified scry path in the `$task` sent to Eyre.  This full path can be used directly to scry a value, and then the `desk` in the path should be used to load the mark conversion to `%mime`.

Reply:
IIUC the paths are already fully-qualified; `+scry-to-mime` calls `de-beam` on it. I've been using `.^(cors-registry:eyre %ex /=//=/cors)` to test the scry functionality, but since the desk is `%$`, scrying the mark conversion fails. On the other hand, if you try `/=base=/cors`, the Eyre scry fails. So I'm not sure what to do here.

Separately, I noticed that the `cage` returned for a (successful) `/cors` scry is `%noun`, not `%cors-registry`. Scrying a `noun -> mime` mark conversion seems to succeed, but the resulting gate behaves weirdly. Should we check explicitly for `%noun`?

Reply:
Let's just rip this functionality out of Eyre for now.  It's not that hard for an agent to scry and mark-convert the result, then ask Eyre to use that as the HTTP response.  Halfway future-proofing by allowing the agent to also specify a scry path doesn't meaningfully improve this system, so I think we should ditch it in favor of deploying something that works sooner.

Don't lose the code, for it, though.  When it comes time to expose a scry namespace in response to HTTP requests, we'll need to think this through.  I'm starting to lean toward having this be Clay functionality: ask Clay in the scry path to mark-convert the file for you.

Reply:
kind of a relief tbh. I kept the tagged union so that we can add a `%scry` case later, is that the right way to go?

Reply:
Yeah that's fine.